probe/endpoint/conntrack.go

```diff
@@ -75,11 +81,11 @@ type Conntracker struct {
 }
 
 // NewConntracker creates and starts a new Conntracter
-func NewConntracker(existingConns bool, args ...string) (*Conntracker, error) {
+var NewConntracker = func(existingConns bool, args ...string) (Conntracker, error) {
```

Comment:
Whats the downsides of this approach?  I'm not a huge fan of the factory style, if only because it pollutes potentially clean APIs (it would work better if golang had optional parameters...)

I guess there are probably cases (this being one) where we would just pass a conntracker to natmapper; that seems clean & obvious, so I'll make that change.